docs/UserGuide.md

```diff
@@ -178,7 +186,7 @@ Format: `shift-add d/DAY t/TIME [r/ROLE NUMBER_NEEDED]...`
 
 * Adds a shift on the specified `DAY` at the specified `TIME`.
 * The day specified should take one of these values: **Mon, Tue, Wed, Thu, Fri, Sat, Sun**. These values are case-
-  insensitive i.e. `Mon`, `MON`, `mon`, `mOn` etc. are all accepted.
+  insensitive i.e. `Mon`, `MON`, `mon` etc. are all accepted.
```

Comment:
May I know the rationale behind this change since this example is used above for `UNAVAILABILITY` as well?

Reply:
I was supposed to delete it for `UNAVAILABILITY` as well but I missed it out; will fix it! Because our days are now enum values so we can't have numbers like 0 inside

Reply:
I believe that's a capital 'O'